Source/Core/DolphinQt2/MenuBar.cpp

```diff
@@ -133,7 +133,7 @@ void MenuBar::AddToolsMenu()
 
   // Label will be set by a NANDRefresh later
   m_boot_sysmenu =
-      AddAction(gc_ipl, QStringLiteral(""), this, [this] { emit BootWiiSystemMenu(); });
+      AddAction(tools_menu, QStringLiteral(""), this, [this] { emit BootWiiSystemMenu(); });
   m_import_backup = AddAction(gc_ipl, tr("Import BootMii NAND Backup..."), this,
```

Comment:
It seems like a lot of things in the tools menu is out of place when compared to WX: 
![image](https://user-images.githubusercontent.com/4411333/30246821-d04a5ac2-95c1-11e7-9a15-2ba97d3c0f99.png)

WX looks like this: 
![image](https://user-images.githubusercontent.com/4411333/30246842-35515b14-95c2-11e7-9f1d-95411c1c94d8.png)